docs/development/codeguide.rst

```diff
@@ -5,64 +5,51 @@
 Coding Guidelines
 *****************
 
-This section describes requirements and guidelines that should be followed
-both for the core package and for affiliated packages.
-
-.. note:: Affiliated packages will only be considered for integration as a
-          module in the core package once these guidelines have been
-          followed.
+This section describes requirements and guidelines that should be followed both
+by the core package and by coordinated packages, and these are also recommended
+for affiliated packages.
 
 Interface and Dependencies
 ==========================
 
-* All code must be compatible with Python 3.5 and later.
-  Usage of ``six``, ``__future__``, and ``2to3`` is not longer acceptable.
+* All code must be compatible with Python 3.6 and later.
```

Comment:
Can we refer to astropy's minimal python version or so?

Reply:
You mean to avoid hard-coding the minimum version? We could link to the ``setup.cfg`` file and point to the ``requires_python`` section?

Reply:
Yes, something like that. I just find that in my own packages I forget to update these parts...

Reply:
Done